elementary/wingpanel-indicator-bluetooth
# Synchronise settings and global "is-powered" state

After merging this, it is necessary to merge https://github.com/elementary/switchboard-plug-bluetooth/pull/206 to allow switchboard to continue to sync properly.

After merging both PRs, all four switches affecting the Bluetooth power state *should* remain synchonised and stable.

To test:

1) Check that changing the Bluetooth enabled state by any of the switches in Wingpanel, Switchboard or `io.elementary.desktop.wingpanel.bluetooth.bluetooth-enabled` results in the other states changing accordingly.

2) Suspend and change the Bluetooth enabled state using the greeter wingpanel.  This should operate stably.  On reawakening, the changed state should be reflected in the session wingpanel.

3) Restart and check greeter Bluetooth indicator works as expected
